sdk/tests/deqp/framework/common/tcuRGBA.js

```diff
@@ -37,7 +37,7 @@ var deMath = framework.delibs.debase.deMath;
      * @constructor
      */
     tcuRGBA.RGBA = function() {
-        /** @type {Uint32Array} */ this.m_value = new Uint32Array(4);
+        /** @type {goog.NumberArray} */ this.m_value;
```

Comment:
Why not just take the goog.NumberArray as argument to the constructor and assign this.m_value from it?